app/views/shared/summary_cards/_list.html.slim

```diff
@@ -4,7 +4,7 @@
       div class="entity-icon entity-icon--lists"
```

Comment:
For some reason, tags aren't showing up in the summary card. I've got them listed here: https://cl.ly/0b3G1K3V272F

But they don't show up in the search results: https://cl.ly/2Y1i3K0G0G3v